qlib/data/data.py

```diff
@@ -478,13 +478,13 @@ def expression_calculator(inst, start_time, end_time, freq, column_names, spans=
 
         data = pd.DataFrame(obj)
         _calendar = Cal.calendar(freq=freq)
-        data.index = _calendar[data.index.values.astype(np.int)]
+        data.index = _calendar[data.index.values.astype(int)]
```

Comment:
Why is this necessary ?

Reply:
`np.int`, and `np.bool` is deprecated in numpy 1.20.0. We can use `int` directly or specify `np.int32` or `np.int64`.